common/test/views/support/cleaner/AmpAdCleanerTest.scala

```diff
@@ -39,7 +39,7 @@ class AmpAdCleanerTest extends FlatSpec with Matchers {
 
   }
 
-  "AmpAdCleaner" should "only add 2 ads in total" in {
+  "AmpAdCleaner" should "only add 2 ads for short article" in {
```

Comment:
assuming this test still passes, something is broken, as the intention was to make sure it didn't add 3 adverts where there was space to.
So you might want to look into that too.